packages/main/src/Bar.js

```diff
@@ -0,0 +1,124 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import BarTemplate from "./generated/templates/BarTemplate.lit.js";
+import BarDesign from "./types/BarDesign.js";
+
+// Styles
+import BarCss from "./generated/themes/Bar.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-bar",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.Bar.prototype */ {
+		/**
+		 * Defines the <code>ui5-bar</code> design.
+		 * <br><br>
+		 * <b>Note:</b> Available options are "BottomBorder", "TopBorder", "AllBorders".
+		 *
+		 * @type {BarDesign}
+		 * @defaultvalue "BottomBorder"
+		 * @public
+		 */
+		design: {
+			type: BarDesign,
+			defaultValue: BarDesign.BottomBorder,
+		},
+	},
+	slots: /** @lends sap.ui.webcomponents.main.Bar.prototype */ {
+		/**
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @since 1.0.0-rc.11
+		 * @public
+		 */
+		startContent: {
+			type: HTMLElement,
+		},
+		/**
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @since 1.0.0-rc.11
+		 * @public
+		 */
+		middleContent: {
+			type: HTMLElement,
+		},
+		/**
+		 * @type {HTMLElement[]}
+		 * @slot
+		 * @since 1.0.0-rc.11
+		 * @public
+		 */
+		endContent: {
+			type: HTMLElement,
+		},
+	},
+	events: /** @lends sap.ui.webcomponents.main.Bar.prototype */ {
+		//
+	},
+};
+
+/**
+ * @class
+ *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
```

Comment:
You can reuse the description (or part of it) from openui5, but we should add some overview and description for the component